src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/core/ConfJsonWrapper.java

```diff
@@ -90,7 +91,7 @@ int getPerVertexBufferLength() {
 
   @VisibleForTesting
   void setMutedRcaList(List<String> mutedRcas) {
-    mutedRcas = mutedRcas;
```

Comment:
let's remove this setMutedRcaList function completely. looks like it is not being used at all

Reply:
It looks like this method was added fairly recently for testing purposes. It is possible that the associated tests have just not been written.
https://github.com/opendistro-for-elasticsearch/performance-analyzer-rca/commit/5c165cd24147abd70cd34ede684a0a534b245c64

Reply:
Let me add joydeep to this for clarity.

Reply:
We can remove it. The test needing it can add it back